spec/mainadditions.html

```diff
@@ -212,91 +208,167 @@ <h1>
-          1. Let _y_ be ? ToNumber(_values_[0]).
-          1. Let _m_ be ? ToNumber(_values_[1]).
-          1. If _numberOfArgs_ > 2, let _dt_ be ? ToNumber(_values_[2]); else let _dt_ be *1*<sub>𝔽</sub>.
-          1. If _numberOfArgs_ > 3, let _h_ be ? ToNumber(_values_[3]); else let _h_ be *+0*<sub>𝔽</sub>.
-          1. If _numberOfArgs_ > 4, let _min_ be ? ToNumber(_values_[4]); else let _min_ be *+0*<sub>𝔽</sub>.
-          1. If _numberOfArgs_ > 5, let _s_ be ? ToNumber(_values_[5]); else let _s_ be *+0*<sub>𝔽</sub>.
-          1. If _numberOfArgs_ > 6, let _milli_ be ? ToNumber(_values_[6]); else let _milli_ be *+0*<sub>𝔽</sub>.
-          1. If _y_ is *NaN*, then
-            1. Let _yr_ be *NaN*.
-          1. Else,
-            1. Let _yi_ be ! ToIntegerOrInfinity(_y_).
-            1. If 0 ≤ _yi_ ≤ 99, let _yr_ be *1900*<sub>𝔽</sub> + 𝔽(_yi_); otherwise, let _yr_ be _y_.
-          1. Let _finalDate_ be MakeDate(MakeDay(_yr_, _m_, _dt_), MakeTime(_h_, _min_, _s_, _milli_)).
-          1. Let _dv_ be TimeClip(UTC(_finalDate_)).
-        1. Let _O_ be ? OrdinaryCreateFromConstructor(NewTarget, *"%Date.prototype%"*, « [[DateValue]] »).
-        1. Set _O_.[[DateValue]] to _dv_.
-        1. Return _O_.
-      </emu-alg>
-    </emu-clause>
-  </emu-clause>
+  <emu-clause id="sec-date-objects">
+    <h1>Date Objects</h1>
 
-  <emu-clause id="sec-temporal-properties-of-the-date-constructor">
-    <h1><a href="https://tc39.es/ecma262/#sec-properties-of-the-date-constructor">Properties of the Date Constructor</a></h1>
+    <emu-clause id="sec-overview-of-date-objects-and-definitions-of-abstract-operations">
+      <h1>Overview of Date Objects and Definitions of Abstract Operations</h1>
 
-    <emu-clause id="sec-temporal-date.now">
-      <h1><a href="https://tc39.es/ecma262/#sec-date.now">Date.now ( )</a></h1>
-      <del class="block">
-        <p>This function returns the time value designating the UTC date and time of the occurrence of the call to it.</p>
-      </del>
-      <ins class="block">
-        <p>This function performs the following steps when called:</p>
+      <p>[...]</p>
+
+      <emu-clause id="sec-time-zone-identifiers">
+        <h1>Time Zone Identifiers</h1>
+
+        <p>
+          Time zones in ECMAScript are represented by <dfn variants="time zone identifier">time zone identifiers</dfn>, which are Strings composed entirely of code units in the inclusive interval from 0x0000 to 0x007F.
+          Time zones supported by an ECMAScript implementation may be <dfn variants="available named time zone">available named time zones</dfn>, represented by the [[Identifier]] field of the Time Zone Identifier Records returned by AvailableNamedTimeZoneIdentifiers, or <dfn variants="offset time zone">offset time zones</dfn>, represented by Strings for which IsTimeZoneOffsetString returns *true*.
+          <ins>Time zone identifiers are compared using ASCII-case-insensitive comparisons.</ins>
```

Comment:
The 262 editors wanted us to move this line into Temporal because it wasn't needed to support current 262 text.